WalletWasabi.Fluent/ViewModels/Wallets/WalletSettingsViewModel.cs

```diff
@@ -11,28 +16,32 @@ public partial class WalletSettingsViewModel : RoutableViewModel
 	[AutoNotify] private bool _autoCoinJoin;
 	[AutoNotify] private int _minAnonScoreTarget;
 	[AutoNotify] private int _maxAnonScoreTarget;
+	[AutoNotify] private string _plebStopThreshold;
+
+	private Wallet _wallet;
 
 	public WalletSettingsViewModel(WalletViewModelBase walletViewModelBase)
 	{
-		var wallet = walletViewModelBase.Wallet;
-		Title = $"{wallet.WalletName} - Wallet Settings";
-		_preferPsbtWorkflow = wallet.KeyManager.PreferPsbtWorkflow;
-		_autoCoinJoin = wallet.KeyManager.AutoCoinJoin;
-		IsHardwareWallet = wallet.KeyManager.IsHardwareWallet;
-		IsWatchOnly = wallet.KeyManager.IsWatchOnly;
+		_wallet = walletViewModelBase.Wallet;
+		Title = $"{_wallet.WalletName} - Wallet Settings";
+		_preferPsbtWorkflow = _wallet.KeyManager.PreferPsbtWorkflow;
+		_autoCoinJoin = _wallet.KeyManager.AutoCoinJoin;
+		IsHardwareWallet = _wallet.KeyManager.IsHardwareWallet;
+		IsWatchOnly = _wallet.KeyManager.IsWatchOnly;
+		_plebStopThreshold = _wallet.KeyManager.PlebStopThreshold.ToString();
```

Comment:
Somehow, one of my wallets Threshold is `null` in the json, so this line throws a huge exception and the app crashes.

```suggestion
		_plebStopThreshold = _wallet.KeyManager.PlebStopThreshold.ToString() ?? KeyManager.DefaultPlebStopThreshold.ToString();
```
Just to be sure?